### DISCUSSION TITLE
Go supported version

### DISCUSSION THREAD

**Discussion by GoLoTrust** [bottlerocket-os/bottlerocket#4188] [Q&A] [answered]
Hi everyone! just a silly question since I'm not a Bottlerocket expert so I wonder what version of Go it's currently supported or where can I find it. Thanks in advance!

**Comment by arnaldo2792** [comment, marked as answer]
Hey there! You can find the current Golang versions in the Bottlerocket SDK:

https://github.com/bottlerocket-os/bottlerocket-sdk/tree/develop/hashes

Ideally, we keep all the binaries in all the variants in sync and use the same Golang version to compile them. However, there are cases where we might have to use a slightly newer versions of Golang due to backwards-incompatibilities.

**Comment by GoLoTrust** [comment]
Hey @arnaldo2792 !! Thank you so much for your response and clarify my question I really appreciate it!!
